dev/archery/archery/crossbow/cli.py

```diff
@@ -233,6 +234,31 @@ def latest_prefix(obj, prefix, fetch):
     click.echo(latest.branch)
 
 
+@crossbow.command()
+@click.argument('job-name', required=True)
+@click.option('--fetch/--no-fetch', default=True,
+              help='Fetch references (branches and tags) from the remote')
+@click.pass_obj
+def save_report_data(obj, job_name, fetch):
+    """
+    Just print there the state of the job
+    """
+    output = obj['output']
+
+    queue = obj['queue']
+    print(dir(queue))
+
+    if fetch:
+        queue.fetch()
+
+    job = queue.get(job_name)
+    report = JsonReport(job=job)
+    
+    with tempfile.NamedTemporaryFile(delete=False) as temp:
+        temp.write(report.show(output))
```

Comment:
I still don't get where `report.show` should come from. I don't see any `show` method implemented in `JsonReporter` and I guess you actually wanted to invoke `report.get_json_tasks()` here as that's what gives you back the json of the report as a string that you can write back to file..